### PR TITLE
NIFI-12725 Update json-schema-validator to 1.3.0

### PR DESCRIPTION
**Pull request by mr1716**
# Summary

[NIFI-12725](https://issues.apache.org/jira/browse/NIFI-12725)

# Tracking

Please complete the following tracking steps prior to pull request creation.

### Issue Tracking

- [X] [Apache NiFi Jira](https://issues.apache.org/jira/browse/NIFI-12725) issue created

### Pull Request Tracking

- [X] Pull Request title starts with Apache NiFi Jira issue number, such as `NIFI-00000`
- [X] Pull Request commit message starts with Apache NiFi Jira issue number, as such `NIFI-00000`

### Pull Request Formatting

- [X] Pull Request based on current revision of the `main` branch
- [X] Pull Request refers to a feature branch with one commit containing changes

# Verification

Please indicate the verification steps performed prior to pull request creation.

### Build

- [ ] Build completed using `mvn clean install -P contrib-check`
  - [ ] JDK 21

### Licensing

- [ ] New dependencies are compatible with the [Apache License 2.0](https://apache.org/licenses/LICENSE-2.0) according to the [License Policy](https://www.apache.org/legal/resolved.html)
- [ ] New dependencies are documented in applicable `LICENSE` and `NOTICE` files

### Documentation

- [ ] Documentation formatting appears as expected in rendered files

### CODE REVIEW

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml [hunk 1]
```diff
@@ -191,7 +191,7 @@
         <dependency>
             <groupId>com.exceptionfactory.socketbroker</groupId>
             <artifactId>socketbroker</artifactId>
-            <version>1.1.0</version>
+            <version>1.3.0</version>
```
Comment (exceptionfactory) [inline nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml:194]: This change is for a different library and should be reverted.

**Review by exceptionfactory — CHANGES_REQUESTED** [review]
Thanks @mr1716, the changeset includes `socketbroker`, which needs to be reverted.

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml (hunk 1 above)
Comment (mr1716) [reply nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml]: @exceptionfactory thanks. Change reverted.

**Review by exceptionfactory — CHANGES_REQUESTED** [review]
Initial build results show test failures for JSON Schema components, so it appears that version 1.3.0 changed something substantively. This will require closer evaluation.

**Comment by exceptionfactory** [comment]
The changelog for version [1.2.0](https://github.com/networknt/json-schema-validator/releases/tag/1.2.0) indicates breaking changes, same for 1.3.0. The `json-schema-validator` does not follow semantic versioning, so this upgrade will require code changes. I am closing this pull request for now, pending further changes necessary to adapt to API changes.

**Pull request closed without merging**